addon/templates/components/polaris-text-field.hbs

```diff
@@ -0,0 +1,118 @@
+<!-- template-lint invalid-interactive=false -->
+
+<div class="Polaris-Labelled__LabelWrapper">
+  <div class="Polaris-Label">
+    <label id="{{id}}Label" for={{id}} class="Polaris-Label__Text">{{label}}</label>
+  </div>
+  {{#if labelAction}}
+    {{polaris-button
+      plain=true
+      text=labelAction.content
```

Comment:
We tend to use `text` instead of `content` owing to possible issues if someone passes an Ember proxy object (which has its own `content` property already)